Workbooks/AKS/Node Disk Capacity AKS/Node Disk Capacity AKS.workbook

```diff
@@ -12,7 +12,54 @@
       "type": 9,
       "content": {
         "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Resource}"
+        ],
         "parameters": [
+          {
+            "id": "fc07cb35-856f-44ce-85a6-02397b06aef3",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resource",
+            "type": 5,
+            "isRequired": true,
+            "value": "value::1",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.containerservice/managedclusters": true,
+                "microsoft.kubernetes/connectedclusters": true,
+                "microsoft.redhatopenshift/openhhiftclusters": true
```

Comment:
There is typo in openhhift. and also we discussed we are not adding openshift resource type , right? so we can remove this.